src/azure-cli/azure/cli/command_modules/acs/_consts.py

```diff
@@ -12,6 +12,13 @@
 CONST_SPOT_EVICTION_POLICY_DELETE = "Delete"
 CONST_SPOT_EVICTION_POLICY_DEALLOCATE = "Deallocate"
 
+# consts for upgrade channel
+CONST_RAPID_UPGRADE_CHANNEL = "rapid"
+CONST_STABLE_UPGRADE_CHANNEL = "stable"
+CONST_PATCH_UPGRADE_CHANNEL = "patch"
+CONST_NODE_IMAGE_UPGRADE_CHANNEL = "node-image"
+CONST_NONE_UPGRADE_CHANNEL = "none"
```

Comment:
If these enumerations are only used in `arg_type=get_enum_type()` and are not used elsewhere, I suggest that they do not need to be defined separately. You can just define it this way https://github.com/Azure/azure-cli/pull/18825/files#r687325877

Reply:
Followed the style of defining the array at the top of the _params file, and using that in the get_enum_type() calls. Curious what you think.

Reply:
I mean, instead of defining these enumerations, we can write valid values directly to `arg_type=get_enum_type()`